### ISSUE TITLE
Request: deleteTable returns TableDescription

### ISSUE THREAD

**Issue by ScottONeal** [awslabs/dynamodb-data-mapper-js#105]
First off, thanks for this library! It creates a nice bridge.

Currently, `deleteTable()` returns `undefined` on success, however, it would be nice to have the response callback of the delete call as it is in the [`aws-sdk deleteTable`](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html#deleteTable-property). This would allow us to run assertions that the correct table was, in fact, deleted.

If this is undesired functionality please feel free to delete the issue. 👍

**Comment by ScottONeal** [comment]
Right, I do follow this and use the successful resolution as you demonstrate. It would just be nice to formally verify the deletion of the correct properties with the reponse of `TableDefinition` from `DynamoDB`. 

Anyways, thanks for your quick response! 🍻

**Comment by jeskew** [comment]
Sorry; I misunderstood the ask in the original post. `deleteTable` is currently discarding the table definition, but that can be changed.
